src/utils/rest-api.ts

```diff
@@ -573,9 +574,10 @@ export function saveCompositeModification(id: string, name: string) {
  * @returns {Promise<Response>}
  */
 export function saveCriteriaBasedContingencyList(id: string, form: CriteriaBasedEditionFormData) {
-    const { name, equipmentType, criteriaBased } = form;
+    const { name, description, equipmentType, criteriaBased } = form;
     const urlSearchParams = new URLSearchParams();
     urlSearchParams.append('name', name);
+    urlSearchParams.append('description', description ?? '');
```

Comment:
Why send description in this case ? If description is undefined, simply ignore it.
```suggestion
if (description) {
        urlSearchParams.append('description', description);
    }
```
(same for the 3 following similar functions)

Reply:
because when you want to delete description, you send an empty one.